### PR TITLE
Overall update

### PR DESCRIPTION
**Pull request by userMacieG**
ox support
code cleanup + reformatted

Working version, this is not a final version.

### CODE REVIEW

File: fxmanifest.lua [hunk 1]
```diff
@@ -8,7 +8,7 @@ shared_scripts {
     'config.lua',
     '@qb-core/shared/locale.lua',
     'locales/en.lua',
-    'locales/*.lua'
```
Comment (YvanCywan) [inline fxmanifest.lua:11]: Why delete this?

This removes support for other languages.

File: client/main.lua [hunk 2]
```diff
@@ -461,7 +516,10 @@ AddEventHandler('QBCore:Client:OnPlayerLoaded', function()
 end)
 
 AddEventHandler("onResourceStart", function(res)
-    if res ~= GetCurrentResourceName() then return end
+    if res ~= GetCurrentResourceName() then
+        return
+    end
```
Comment (YvanCywan) [inline client/main.lua:521]: I don't necessarily see the point on expanding easy and clear 1 liners.

File: client/main.lua [hunk 3]
```diff
@@ -402,23 +448,29 @@ local function enterVehicle(veh, indexgarage, type, garage)
 end
 
 local function CreateBlipsZones()
-    if blipsZonesLoaded then return end
+    if blipsZonesLoaded then
+        return
+    end
```
Comment (YvanCywan) [inline client/main.lua:453]: I don't necessarily see the point on expanding easy and clear 1 liners.

**Review by YvanCywan — CHANGES_REQUESTED** [review]

File: server/main.lua [hunk 4]
```diff
@@ -291,13 +333,15 @@ QBCore.Functions.CreateCallback('qb-garage:server:GetPlayerVehicles', function(s
                     v.state = Lang:t("status.impound")
                 end
 
-                local fullname
+                local fullname = "Unknown"
```
Comment (D4isDAVID) [inline server/main.lua:336]:
```suggestion
                local fullname
```
This value isn't used.

File: client/main.lua [hunk 5]
```diff
@@ -295,45 +305,63 @@ RegisterNetEvent("qb-garages:client:VehicleList", function(data)
...
             end
 
             MenuGarageOptions[#MenuGarageOptions + 1] = {
-                header = leave,
-                txt = "",
-                params = {
-                    event = "qb-menu:closeMenu",
-                }
+                title = leave,
+                onSelect = function(_)
```
Comment (D4isDAVID) [inline client/main.lua:354]:
```suggestion
                onSelect = function()
```
The `_` isn't needed in functions where no parameters are used at all.

File: client/main.lua [hunk 6]
```diff
@@ -13,52 +12,46 @@ local garageZones = {}
...
                 args = {
                     type = type,
                     garage = garage,
-                    index = indexgarage,
+                    index = indexgarage
                 }
+            },
+            {
+                title = leave,
+                onSelect = function(_)
```
Comment (D4isDAVID) [inline client/main.lua:48]:
```suggestion
                onSelect = function()
```
Ditto

**Review by D4isDAVID — CHANGES_REQUESTED** [review]

File: client/main.lua (hunk 3 above)
Comment (YvanCywan) [reply client/main.lua:453]: Unresolving this as there is no clear answer  and it hasn't been changed.

File: client/main.lua (hunk 2 above)
Comment (YvanCywan) [reply client/main.lua:521]: Unresolving this as there is no clear answer and it hasn't been changed.

File: fxmanifest.lua (hunk 1 above)
Comment (D4isDAVID) [reply fxmanifest.lua:11]: Resolving since it was added back.

**Pull request closed without merging**